package.json

```diff
@@ -36,7 +36,7 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-stage-1": "^6.5.0",
     "babel-register": "^6.9.0",
-    "browser-sync": "^2.13.0",
+    "browser-sync": "^2.18.5",
```

Comment:
Why are we raising the browser-sync and node-sass version numbers? We have to be a little careful when updating packages since it may break existing stuff (and I'm not sure what these packages have to do with this request).